### PR TITLE
Vue.js plugin for iMask

### PR DESCRIPTION
**Pull request by naprimer**

### CODE REVIEW

**Comment by uNmAnNeR** [comment]
merged

File: plugins/vue/src/index.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import VueTypes from 'vue-types';
```
Comment (c01nd01r) [inline plugins/vue/src/index.js]: Why is not built-in props validation used?
https://vuejs.org/v2/guide/components.html#Prop-Validation

File: plugins/vue/src/index.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  normalizeZeros: VueTypes.bool,
+  padFractionalZeros: VueTypes.bool,
+
+  // dynamic
+  dispatch: VueTypes.func
+};
+
+const component = {
+  name: 'MaskedInput',
+  template: '<input>',
```
Comment (c01nd01r) [inline plugins/vue/src/index.js]: This will not work with Vue runtime build.
https://github.com/vuejs/vue/tree/dev/dist#runtime--compiler-vs-runtime-only

**Comment by uNmAnNeR** [comment]
@c01nd01r Yes, you are right. I fixed it during merge, please take a look at actual sources.

**Pull request closed without merging**